build/reports/wiki_home.Rmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "History of published images"
+date: "`r format(Sys.time(), '%Y-%m-%d %H:%M:%S %Z')`"
+output:
+  github_document:
+    toc: false
+    df_print: kable
+    html_preview: false
+params:
+  git_repository: ""
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE)
+knitr::opts_knit$set(root.dir = rprojroot::find_root_file(criterion = rprojroot::is_git_root))
+```
+
+```{r data_manipulation}
+df_args <- jsonlite::read_json("build/args.json") |>
+  tibble::enframe(name = "src") |>
+  dplyr::mutate(order_number = dplyr::row_number()) |>
```

Comment:
It does not appear that this process should be done at this stage.
![image](https://user-images.githubusercontent.com/50911393/212485434-5a8b3cf3-ce0e-4c48-9ea3-90ea5f79186a.png)